integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/ClusterCreationTest.java

```diff
@@ -29,9 +29,10 @@ public void createClusterWithoutNetworkAndPlacementShouldBadRequestException(Moc
                 .given(StackTestDto.class)
                 .withEmptyNetwork()
                 .withEmptyPlacement()
-                .when(StackTestAction::create, key("error"))
-                .expect(BadRequestException.class, key("error")
-                        .withExpectedMessage("Network must be specified!"))
+                .whenException(StackTestAction::create, BadRequestException.class,
+                        key("error")
```

Comment:
I think, in this case, we don't need the key. But it won't cause any error.

Reply:
This will be the `key` at `LOGGER.info("when {} action on {} by {}, name: {}", key, entity, who, entity.getName());`. So we should change this key to a more meaningful one, for example: **badRequest** or something similar.

Thank you for pointing this out!

Reply:
Yes, you are right. Thanks.